### DISCUSSION TITLE
Shit, what if we create TeleDrive as an installable desktop app? 🤯

### DISCUSSION THREAD

**Discussion by mgilangjanuar** [mgilangjanuar/teledrive#47] [Ideas]
If we build this as a desktop app we don't need to have a server and operational costs 🤣😱

**Comment by mgilangjanuar** [comment]
limitations & questions:

 - public files that need a public URL should have a server for it
 - running database, it should be in the server too
 - how to get secret env from an application? 🤔

**Comment by izznfkhrlislm** [comment]
Nice idea to try out Flutter desktop

**Reply by mgilangjanuar** [reply]
Good idea, I'll update the API in Postman too 🤣
